### PR TITLE
Convert remaining UntypedActor in docs

### PR DESCRIPTION
**Pull request by ortibazar**
Refs  #22182

### CODE REVIEW

**Comment by ktoso** [comment]
OK TO TEST

**Comment by ktoso** [comment]
Thanks a lot :)

File: akka-docs/rst/java/persistence.rst [hunk 1]
```diff
@@ -125,8 +125,8 @@ It contains instructions on how to run the ``PersistentActorExample``.
   It's also possible to switch between different command handlers during normal processing and recovery
   with ``getContext().become()`` and ``getContext().unbecome()``. To get the actor into the same state after
   recovery you need to take special care to perform the same state transitions with ``become`` and
-  ``unbecome`` in the ``receiveRecover`` method as you would have done in the command handler.
-  Note that when using ``become`` from ``receiveRecover`` it will still only use the ``receiveRecover``
+  ``unbecome`` in the ``createReceiveRecover`` method as you would have done in the command handler.
+  Note that when using ``become`` from ``createReceiveRecover`` it will still only use the ``createReceiveRecover``
```
Comment (patriknw) [inline akka-docs/rst/java/persistence.rst]: good catch

**Review by patriknw — APPROVED** [review]
LGTM, thank you so much @ortigali

**Pull request merged**